docs/pkgdown.yml

```diff
@@ -2,5 +2,5 @@ pandoc: 2.16.2
 pkgdown: 1.6.1
 pkgdown_sha: ~
 articles: {}
-last_built: 2021-11-30T16:12Z
+last_built: 2021-11-30T18:05Z
```

Comment:
Probably a good idea to re-run pkgdown as it will automagically pick up the vignettes.